lib/exercises.rb

```diff
@@ -1,19 +1,40 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n log n) - ruby's quicksort is O(n log n)
+# Space Complexity: O(n) - creation of the anagrams hashmap
 
 def grouped_anagrams(strings)
```

Comment:
The time complexity is O(n * m log m) where m is the length of the strings, unless you know that the strings are within a set size, in which case you can call it O(n).